snoop/data/analyzers/pgp.py

```diff
@@ -1,18 +1,21 @@
+from pathlib import Path
 import subprocess
 from django.conf import settings
 from ..tasks import ShaormaBroken
 
+gpghome = Path(settings.SNOOP_COLLECTION_ROOT).parent / 'gpghome'
```

Comment:
The default gpg home dir is `gnupg` (see https://github.com/liquidinvestigations/node/blob/master/templates/collection.nomad#L201). This is related to https://github.com/liquidinvestigations/node/pull/116

Reply:
Oh, I totally missed that line, thanks! But I still think `gpghome` should be in the collection folder, it just makes more sense.